docs/cloud/dev/spryker-cloud-commerce-os/multi-store-setups.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Multi-store setups
+description: Types of multi-store setups in Spryker Cloud Commerce OS
+template: concept-topic-template
+---
+
+This document describes the multi-store setups supported by Spryker Cloud Commerce OS.
+
+Currently, the following setups are available:
+
+* Separated: a codebase with dedicated databases per store.
+* Shared: a codebase with all the stores sharing a database.
+
+
+## Shared setup
+
+With the shared setup, all the stores share a single database and codebase.
+
+
+![shared setup diagram](https://spryker.s3.eu-central-1.amazonaws.com/docs/cloud/spryker-cloud-commerce-os/multi-store-setups.md/shared-setup.png)
+
+
+### Shared setup: When to use
+
+
+We recommend this setup for the stores that can be described as follows:
+
+* All the stores are united into a multi-store system.
+
+* All the stores follow the same business logic.
+
+* Insignificant differences are covered in the code, for example, using code-buckets.
+
+* Stores are scaled together and don't need different deployment workflows.
+
+### Shared setup: Advantages
+
+The shared setup features:
+
+* Products, customers, orders, and so on are stored in the same database, which simplifies collaborative management.
```

Comment:
It's possible to have fully separated data set per each store, like Products, customers, orders, and so on, which simplifies collaborative management?

Reply:
Are you sure this is applicable to the _shared_ setup?